var/spack/repos/builtin/packages/easybuild/package.py

```diff
@@ -15,6 +15,7 @@ class Easybuild(PythonPackage):
     pypi = "easybuild/easybuild-4.0.0.tar.gz"
     maintainers("boegel")
 
+    version("4.7.0", sha256="e68c280e3a508965aefbdf234872919f777f739cf4787cb11bb97a56e0cf74a6")
```

Comment:
I *think* that `@4.7.0` also needs to be added in the `for` loop below, where the dependencies on `py-easybuild-*` are declared (and hence those packages need an update too)

Reply:
Added! Thanks for noticing this!